packages/zowe-explorer/__tests__/__unit__/trees/dataset/DatasetActions.unit.test.ts

```diff
@@ -240,7 +240,7 @@ describe("Dataset Actions Unit Tests - Function createMember", () => {
             session: blockMocks.session,
         });
         const nonFavoriteLabel = parent.label;
-        parent.label = `${parent.label}`;
+        parent.label = parent.label?.toString();
```

Comment:
This is just a unit test, so not explicitly requesting changes for this - but is line 243 necessary?
Is this a part of resolving "unknown template expression" lint warnings?

Reply:
will resolve in the follow up PR (#3021) 🙏🏽

Reply:
resolved in c610849b919f0f179fafed3f0c4552254263e67e